app/views/content/is-teaching-right-for-me/physics/_article.html.erb

```diff
@@ -0,0 +1,145 @@
+<div class="row inset">
+<section class="col col-720">
+  <p>
+    Teaching physics is an exciting and rewarding career. Schools need more specialist physics teachers, so it's a great choice of subject.
+  </p>
+
+  <p>
+You could inspire your pupils to consider rewarding careers from tackling climate change to artificial intelligence.
+  </p>
```

Comment:
@caitlin-reidy Same as the maths page. Do you think it's worth keeping in the line about £30k for bursaries and scholarships in the intro? Marios said in UR feedback was that it could be a deciding factor to someone choosing physics. So it might be good to mention at the start to encourage people to read on?

Reply:
@caitlin-reidy if we put the yellow box back in for now (Gemma commented about putting it back in for maths) we'd need to have a new 2nd sentence in there so we're not repeating it as it's now in the intro text.